src/utils.codec.ts

```diff
@@ -54,6 +54,6 @@ export function bigIntToHex(value: BigNumber.Value): string {
 }
 
 export function addressToHex(address: Address): string {
-    const buffer = Address.fromBech32(address.toString()).pubkey();
+    const buffer = Address.newFromBech32(address.toString()).getPublicKey();
     return buffer.toString("hex");
```

Comment:
I think this is not needed since an Address can be eaasily converted to hex string. If still needed perhaps simply return `address.toHex()`?